build.gradle

```diff
@@ -83,9 +89,9 @@ task copyToMods(type: Copy) {
 
 task runGameWithBuild(type: Exec) {
     dependsOn copyToMods
-    isWindows() ? commandLine('cmd', '/c', 'starsector.bat') : commandLine('sh', './starsector.sh')
+    isWindows() ? commandLine('cmd', '/c', gameLauncher) : commandLine(gameLauncher)
```

Comment:
Just to note that with this change you are removing the option for debug options that could be defined in starsector.bat file in windows.

Reply:
Could put `starsector-core/starsector.bat` in `game.launcher`? The same log lands in `starsector.log` anyway - if you need to connect with debugger etc, then you probably know what to do to modify that (and this change will not ruin your life).

Reply:
Does not work sadly. I'll leave it for now.